### PR TITLE
Feat/run calculation

### PR DESCRIPTION
**Pull request by mariuswinger**
## Why is this pull request needed?

Need to get results somehow.

## What does this pull request change?

Adds onClick function to run button saving results to state.

## Issues related to this change:
#49

### CODE REVIEW

File: web/src/components/feature/CalculateFluid.tsx [hunk 1]
```diff
@@ -34,23 +44,50 @@ export const CalculateFluid = ({ edit }: { edit: () => void }) => {
       <FlexContainer>
         <TextField
           id="temperature-input"
-          placeholder="15"
+          min={-273.15}
+          value={temperature.toString()}
           label="Temperature"
           unit="ºC"
           type="number"
+          onChange={(event: { target: { value: string } }) =>
```
Comment (soofstad) [inline web/src/components/feature/CalculateFluid.tsx:52]: Think the correct type to use on these are; React.ChangeEvent<HTMLInputElement>

**Review by soofstad — APPROVED** [review]

File: api/src/features/component/component_feature.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 from common.responses import create_response
 from entities.ComponentResponse import ComponentResponse
 
-router = APIRouter(tags=["component"])
+router = APIRouter(tags=["mercury", "component"])
```
[the remark this answers is not in the document]
Comment (mariuswinger) [reply api/src/features/component/component_feature.py:8]: Yes, can do that.

File: web/src/pages/Main.tsx [hunk 3]
```diff
@@ -3,17 +3,43 @@ import { useContext, useState } from 'react'
...
 import { FluidDialog } from '../components/feature/FluidDialog'
+import { Multiflash, MultiflashResponse } from '../api/generated'
+import { AxiosError, AxiosResponse } from 'axios'
+import MercuryAPI from '../api/MercuryAPI'
 
 export const MainPage = (): JSX.Element => {
   const { token } = useContext(AuthContext)
   const [isOpen, setIsOpen] = useState<boolean>(false)
+  const mercuryApi = new MercuryAPI('')
+  const [previousResult, setPreviousResult] = useState<MultiflashResponse>({
```
[the remark this answers is not in the document]
Comment (mariuswinger) [reply web/src/pages/Main.tsx:14]: Yes.

File: web/src/pages/Main.tsx [hunk 4]
```diff
@@ -3,17 +3,22 @@ import { useContext, useState } from 'react'
...
+    componentFractions: {},
+  })
 
   const toggleFluidDialog = () => setIsOpen(!isOpen)
 
   return (
     <>
       <Header />
-      <CalculateFluid edit={toggleFluidDialog} />
+      <CalculateFluid edit={toggleFluidDialog} run={setResult} />
```
Comment (soofstad) [inline web/src/pages/Main.tsx:21]: run property name now seems missleading

File: web/src/components/feature/CalculateFluid.tsx [hunk 5]
```diff
@@ -34,23 +47,74 @@ export const CalculateFluid = ({ edit }: { edit: () => void }) => {
...
+    pressure: pressure,
+  }
+
+  const calculate = (
+    <Button
+      onClick={() => {
+        mercuryApi
+          .computeMultiflash({ multiflash: multiflashInput })
+          .then((result: AxiosResponse<MultiflashResponse>) => {
+            console.log(JSON.stringify(result.data))
```
Comment (soofstad) [inline web/src/components/feature/CalculateFluid.tsx:99]:
```suggestion
```

**Review by soofstad — APPROVED** [review]
Nice! Consider renaming "run" to "setResult" or something

**Pull request merged**